CHANGELOG.md

```diff
@@ -5,27 +5,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.7.0 (Unreleased)
 
 ### Added
 
+- [#1661](https://github.com/openmls/openmls/pull/1661): Add `member_at` getter method to `MlsGroup`, `not_before` and `not_after` getter methods to `Lifetime` and `life_time` getter method to `KeyPackage`.
 - [#1688](https://github.com/openmls/openmls/pull/1688): Add `unknown()` getter method to `Extensions`.
 - [#1666](https://github.com/openmls/openmls/pull/1666): Add `members()` and `group_context()` getter methods to `StagedWelcome`.
 - [#1672](https://github.com/openmls/openmls/pull/1672): Add `epoch()` getter method to `VerifiableGroupInfo`.
 - [#1673](https://github.com/openmls/openmls/pull/1673): Return more specific error when attempting to decrypt own messages: `ProcessMessageError::ValidationError(ValidationError::CannotDecryptOwnMessage)`.
+- [#1675](https://github.com/openmls/openmls/pull/1675): Add `CommitBuilder` that can be used to create commit messages.
+- [#1682](https://github.com/openmls/openmls/pull/1682): Add stage provider backed by Sqlite.
+- [#1704](https://github.com/openmls/openmls/pull/1704): Add support for SelfRemove proposals as specified in the MLS extensions draft specification.
```

Comment:
Let's add a link to https://datatracker.ietf.org/doc/html/draft-ietf-mls-extensions